dev/TabView/TabView.cpp

```diff
@@ -639,6 +643,15 @@ void TabView::OnItemsChanged(winrt::IInspectable const& item)
             {
                 newItem->OnTabViewWidthModeChanged(TabWidthMode());
                 newItem->SetParentTabView(*this);
+
+                // Item was added as a result of the AddTab button being pressed.
+                // To allow users using narrator know of the new item, move focus to item.
+                if (moveFocusToLastAddedItem)
+                {
+                    // Not sure why, but only both lines in combination give us the wanted effect
+                    winrt::FocusManager::TryMoveFocus(winrt::FocusNavigationDirection::Left);
+                    winrt::FocusManager::TryFocusAsync(*newItem, winrt::FocusState::Programmatic);
```

Comment:
This seems off. Can you do  (newItem as Control).Focus instead ?

Reply:
Tried `newItem.try_as<winrt::Control>().Focus(winrt::FocusState::Keyboard);` and it didn't move focus. It seems only calling both lines results in the desired behavior.